wandb/apis/public.py

```diff
@@ -452,7 +454,7 @@ def update(self):
         %s
         ''' % RUN_FRAGMENT)
         res = self._exec(mutation, id=self.id, tags=self.tags,
-                         description=self.description, config=self.json_config)
+                         description=self.description, notes=self.notes, display_name=self.display_name, config=self.json_config)
```

Comment:
I wonder what the backend does when it gets both description and notes...

Reply:
It is handled intelligently, preferring displayname and notes

If you set description it will generate displayname and notes
If displayname or notes is set, it will generate the description, and overwrite displayname and notes which was set by description